doc/geany.txt

```diff
@@ -2681,6 +2681,9 @@ indent_hard_tab_width             The size of a tab character. Don't change    8
 editor_ime_interaction            Input method editor (IME)'s candidate        0           to new
                                   window behaviour. May be 0 (windowed) or                 documents
                                   1 (inline)
+scrollwheel_lines                 Number of lines the scrollwheel scrolls      3           immediately
```

Comment:
First: why?
Second: this is NOT the current value (which is 4, or a system value). Also, as suggested in the previous sentence, Scintilla's default can depend on the system configuration (currently, it does on Windows), so it's not necessarily a great idea to force another value, at least not with a good rationale.
And, why change it from 4 to 3? or is the Windows's default 3? :)

Reply:
I guess I see the reason why you added this: because you need a value when transforming <kbd>Ctrl+Scroll</kbd> to a "regular" scroll event, right?  And then instead of hard-coding it, you added a setting.
I guess that's a fair reason, but there's sill the issues mentioned above.  And is it important that <kbd>Ctrl+Scroll</kbd> scrolls, or could it just do nothing?

At any rate, at least the default should be the same as the current one.  I don't care as much for Windows I admit, but it's probably easy enough to mimic what Scintilla does there as well, isn't it?  The semantic of the option becomes a bit hairy though, because how to conceal a dynamic system value with a user setting?  I guess it could be `0` means use system default (and fallback to 4), and any other value this amount of lines or something like that…

Reply:
I miscounted and the historical value when mouse wheels were introduced to consumers was 3.  So I didn't bother recounting to confirm.  Would be better to use the system default, but I don't know how.